assertj-core/src/test/java/org/assertj/core/condition/NestableCondition_matches_Test.java

```diff
@@ -75,4 +79,35 @@ void should_not_match_if_any_condition_in_nested_level_does_not_match() {
     // THEN
     then(condition.matches(boris)).isFalse();
   }
+
+  @Test
+  void should_accept_conditions_on_supertypes() {
+    // GIVEN
+    final ValueCustomer boris = new ValueCustomer(new Name("Boris", "Johnson"),
+                                                  new Address("10, Downing Street",
+                                                              "SW1A 2AA",
+                                                              new Country("United Kingdom")),
+                                                  12);
+    Condition<ValueCustomer> valueCustomer = nestable("value customer", name(
+                                                                         first("Boris")),
+                                                  value(12));
```

Comment:
It would be nice if the lines could be wrapped. For instance, put `new ValueCustomer` on a new line, and put `new Name` on a new line as well. Currently, the code contains lots of whitespace, and it is hard to follow.

Reply:
I agree, but that is the format I get applying the Eclipse format.

Reply:
Tried to improve it.